api/cpp/include/slint_color.h

```diff
@@ -117,22 +130,40 @@ class Color
     }
 
     /// Converts this color to an RgbaColor struct for easy destructuring.
-    inline RgbaColor<uint8_t> to_argb_uint() const;
+    [[nodiscard]] inline RgbaColor<uint8_t> to_argb_uint() const;
 
     /// Converts this color to an RgbaColor struct for easy destructuring.
-    inline RgbaColor<float> to_argb_float() const;
+    [[nodiscard]] inline RgbaColor<float> to_argb_float() const;
+
+    /// Construct a color from the HSV color space component.
+    /// The hue is comprised between 0 and 360, and the other parameter between 0 and 1
```

Comment:
Use simpler english:

```suggestion
    /// The hue is expected to be in the range between 0 and 360, and the other parameters between 0 and 1.
```